support/gecko/nsstring/src/lib.rs

```diff
@@ -387,7 +387,7 @@ macro_rules! define_string_types {
                     // data.
                     let this: &$StringRepr = mem::transmute(self);
                     if this.data.is_null() {
-                        debug_assert!(this.length == 0);
+                        debug_assert_eq!(this.length, 0);
```

Comment:
This is technically external code, but we'll re-sync anyway, so it's not a big deal.